src/milind/WordFromArrayMaxSumOfNumber.java

```diff
@@ -0,0 +1,30 @@
+package milind;
+
+public class WordFromArrayMaxSumOfNumber {
+	
+	void printWordHavingMaxSumOfNumber(String[] input) {
+		int maxSum = 0;
+		String maxSumWord = "";
+		for(int index = 0; index < input.length; index++) {
+			int sum = 0;
+			String word = input[index];
+			for(int innerIndex = 0; innerIndex < word.length(); innerIndex++) {
+				char ch = word.charAt(innerIndex);
+				if(Character.isDigit(ch)) {
+					sum = sum + ch;

```

Comment:
This is not expected output, did you run your code?
you need to use belo method to get numeric values
Character.getNumericValue(ch)